### PR TITLE
Issue template, mostly for bug reports

### PR DESCRIPTION
**Pull request by kba**
An [issue template](https://help.github.com/articles/creating-an-issue-template-for-your-repository/) to make reporting bugs/problems/feature requests more structured.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+## Steps to Reproduce (for bugs)
+<!--- Provide the input image(s) and list the commands you entered. -->
+1.
+2.
+3.
+4.
+
+## Your Environment
+<!--- Include as many relevant details about the environment you experienced the bug in -->
+<!--- Run `git rev-parse HEAD` in the ocropy directory -->
```
Comment (zuphilip) [inline .github/ISSUE_TEMPLATE.md]: I have never used the `git rev-parse` command so far. Maybe, it is easier to suggest `git log -n 1` where the user might easier see if the version is outdated.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+* Git revision:
+* Operating System and version:
```
Comment (zuphilip) [inline .github/ISSUE_TEMPLATE.md]: I think we should also include the python version here. The numpy, scipy, ... version could also be interesting, but I guess that is too much to ask for the default.

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+<!--- Provide the input image(s) and list the commands you entered. -->
+1.
+2.
+3.
+4.
+
+## Your Environment
+<!--- Include as many relevant details about the environment you experienced the bug in -->
+<!--- Run `git rev-parse HEAD` in the ocropy directory -->
+* Git revision:
```
Comment (zuphilip) [inline .github/ISSUE_TEMPLATE.md]: Maybe add "of ocropy" here.

**Comment by kba** [comment]
Okay, let's try this. If it turns out to be too complicated or not helpful we can always change/disable later.

**Comment by zuphilip** [comment]
Thank you for working on this! Let us see now how this will be used.

**Pull request merged**